rt-proj-02/src/core/camera.h

```diff
@@ -20,6 +20,48 @@ class Camera{
     virtual Ray generate_ray(int x, int y) = 0;
 };
 
+class OrthographicCamera : public Camera {
```

Comment:
As we are doing everything in the orthographic.h and .cpp files, we don't need this declaration here anymore.